data/resources/style.css

```diff
@@ -251,27 +253,32 @@ messagemedia overlay > messageindicators, messagesticker messageindicators {
 }
 
 .sender-text-cyan {
-  color: #2996ad;
+    color: #2996ad;
```

Comment:
```suggestion
  color: #2996ad;
```

Reply:
Can I set css indent_size to 2 in `.editor_config`?

![image](https://user-images.githubusercontent.com/63008755/201970624-0271b68f-cec8-4ff6-a5bf-e9a12e3bdf52.png)